pkgs/by-name/po/poetry/package.nix

```diff
@@ -17,14 +17,27 @@ let
       # We keep the override around even when the versions match, as
       # it's likely to become relevant again after the next Poetry update.
       poetry-core = super.poetry-core.overridePythonAttrs (old: rec {
```

Comment:
I guess `rec` is not needed here?

Reply:
It is, for `version`.

Reply:
Fair enough... but can't we use `old.version` ?

Reply:
No, we're overriding `version`.

Reply:
Ooooh got it, thanks for the enlightenment !